src/Core/Models/Api/Request/CipherRequestModel.cs

```diff
@@ -20,6 +20,7 @@ public class CipherRequestModel
         public string OrganizationId { get; set; }
         public string FolderId { get; set; }
         public bool Favorite { get; set; }
+        public bool PasswordPrompt { get; set; }
```

Comment:
Is this the right name? According to the task, this could be password, PIN, or biometrics. Maybe just "reprompt"?